ext/url/lib.rs

```diff
@@ -148,13 +152,14 @@ fn as_u32_slice(slice: &mut [u8]) -> &mut [u32] {
   }
 }
 
-#[op]
+#[op2(fast)]
+#[smi]
 pub fn op_url_reparse(
   state: &mut OpState,
-  href: String,
-  setter: u8,
-  setter_value: String,
-  buf: &mut [u8],
+  #[string] href: String,
+  #[smi] setter: u8,
+  #[string] setter_value: String,
+  #[buffer] buf: &mut [u8],
```

Comment:
This is actually a legit bug -- the buffer is Uint32Array on the JS side.

Reply:
Ah, nice catch, that's probably why the tests are failing.

Reply:
Updated to main -- will try again